From 454bf582614b13610d1395c2620fa79c145d0d06 Mon Sep 17 00:00:00 2001
From: Wincent Colaiuta <win@wincent.com>
Date: Fri, 27 Feb 2009 15:42:22 +0100
Subject: [PATCH] Use same contents for ignored files in implicit multipart test

For consistency with the other ignored files in the implicit multipart
test, use the same contents in:

  implicitly_multipart_example.text.html.erb~

As in:

 implicitly_multipart_example.ignored.erb
 implicitly_multipart_example.rhtml.bak

This might help avoid confusion like that which prompted me to file
ticket #2030.

Signed-off-by: Wincent Colaiuta <win@wincent.com>
---
 .../implicitly_multipart_example.text.html.erb~    |   11 +----------
 1 files changed, 1 insertions(+), 10 deletions(-)

diff --git a/actionmailer/test/fixtures/test_mailer/implicitly_multipart_example.text.html.erb~ b/actionmailer/test/fixtures/test_mailer/implicitly_multipart_example.text.html.erb~
index 946d99e..6940419 100644
--- a/actionmailer/test/fixtures/test_mailer/implicitly_multipart_example.text.html.erb~
+++ b/actionmailer/test/fixtures/test_mailer/implicitly_multipart_example.text.html.erb~
@@ -1,10 +1 @@
-<html>
-  <body>
-    HTML formatted message to <strong><%= @recipient %></strong>.
-  </body>
-</html>
-<html>
-  <body>
-    HTML formatted message to <strong><%= @recipient %></strong>.
-  </body>
-</html>
+Ignored when searching for implicitly multipart parts.
-- 
1.6.1.3

